lib/livebook/config.ex

```diff
@@ -35,128 +35,60 @@ defmodule Livebook.Config do
   end
 
   @doc """
-  Returns the list of currently available file systems.
+  Returns the local filesystem.
   """
-  @spec file_systems() :: list(FileSystem.t())
-  def file_systems() do
-    Application.fetch_env!(:livebook, :default_file_systems) ++
-      Enum.map(storage().all(:filesystem), &storage_to_fs/1)
+  @spec local_filesystem() :: FileSystem.t()
+  def local_filesystem do
+    :persistent_term.get(:livebook_local_filesystem)
   end
 
   @doc """
-  Appends a new file system to the configured ones.
+  Returns the local filesystem home.
   """
-  @spec append_file_system(FileSystem.t()) :: list(FileSystem.t())
-  def append_file_system(%FileSystem.S3{} = file_system) do
-    attributes =
-      file_system
-      |> FileSystem.S3.to_config()
-      |> Map.to_list()
-
-    storage().insert(:filesystem, generate_filesystem_id(), [{:type, "s3"} | attributes])
-
-    file_systems()
-  end
-
-  @doc """
-  Removes the given file system from the configured ones.
-  """
-  @spec remove_file_system(FileSystem.t()) :: list(FileSystem.t())
-  def remove_file_system(file_system) do
-    storage().all(:filesystem)
-    |> Enum.find(&(storage_to_fs(&1) == file_system))
-    |> case do
-      %{id: id} -> storage().delete(:filesystem, id)
-    end
-
-    file_systems()
+  @spec local_filesystem_home() :: FileSystem.File.t()
+  def local_filesystem_home do
+    FileSystem.File.new(local_filesystem())
   end
 
   @doc """
-  Returns the default directory.
+  Returns the home path.
   """
-  @spec default_dir() :: FileSystem.File.t()
-  def default_dir() do
-    [file_system | _] = Livebook.Config.file_systems()
-    FileSystem.File.new(file_system)
+  @spec home() :: String.t()
+  def home do
+    Application.get_env(:livebook, :home) || System.user_home() || File.cwd!()
```

Comment:
What about `home_path` here to clarify it's a path and then `home` as the `FileSystem.File.t()` version.

Reply:
I am going for symmetry with the OS env var/System.user_home here. I would prefer to keep `Livebook.Config.local_filesystem_home()` long, because it should rarely be used (and it will be removed once we make the default filesystem configurable). :)